app/javascript/packs/ajax.js

```diff
@@ -30,7 +36,7 @@ global.ajaxCall = (
   /** Special handling of all /map/route paths so that instead of loading another page,
    * the route navigation screen is shown on in the browse outlet so that a navigation can be started.
    */
-  if (target === "/route") {
+   if (target === "/route") {
```

Comment:
I suspect this is a space too much 😅 
```suggestion
  if (target === "/route") {
```